### PR TITLE
fix(manage-merge-queue): fix PR removal from queue

### PR DESCRIPTION
**Pull request by danadajian**
### :pencil: Description
- Should fix an issue where the `removePrFromQueue()` function was not being called and thus PRs were not being removed from the queue in certain scenarios.

### CODE REVIEW

File: src/helpers/manage-merge-queue.ts [hunk 1]
```diff
@@ -116,7 +116,7 @@ const getQueuedPullRequests = async (): Promise<PullRequestList> => {
 
 export const enableAutoMerge = async (pullRequestId: string, mergeMethod = 'SQUASH') => {
   try {
-    return await octokitGraphql(`
+    await octokitGraphql(`
```
Comment (PoojaDonekal1294) [inline src/helpers/manage-merge-queue.ts:119]: Why are we removing `return` here?

Comment (danadajian) [reply src/helpers/manage-merge-queue.ts:119]: Just wanted to remove cases of `return await` since it's kind of confusing to me 😂

Comment (PoojaDonekal1294) [reply src/helpers/manage-merge-queue.ts:119]: If we are not using the response returned, then yeah getting rid of `return` makes sense.

**Pull request merged**